packages/components/src/components/post-popovercontainer/post-popovercontainer.tsx

```diff
@@ -69,13 +62,30 @@ export class PostPopovercontainer {
    */
   @Prop() readonly arrow?: boolean = false;
 
+  /**
+   * Enables a safespace through which the cursor can be moved without the popover being disabled
+   */
+  @Prop() readonly safeSpace?: 'triangle' | 'trapezoid';
+
+  // New method for safe space
+  private mouseTrackingHandler = (event: MouseEvent) => {
+    if (!this.safeSpace || !this.host.matches(':where(:popover-open, .popover-open)')) return;
+
+    this.host.style.setProperty('--safe-space-cursor-x', `${event.clientX}px`);
+    this.host.style.setProperty('--safe-space-cursor-y', `${event.clientY}px`);
+  };
+
   componentDidLoad() {
     this.host.setAttribute('popover', '');
     this.host.addEventListener('beforetoggle', this.handleToggle.bind(this));
+    window.addEventListener('mousemove', this.mouseTrackingHandler);
```

Comment:
Only add this listener if there's a `safe-space="triangle"` requested, otherwise it's there for nothing. Ideally the event listener is being added on open and removed on close.